tests/ui_tools/test_buttons.py

```diff
@@ -96,6 +96,20 @@ def test__decode_stream_data(self, stream_data, expected_response):
 
         assert return_value == expected_response
 
+    @pytest.mark.parametrize('pm_data, expected_response', [
+            ('1,2-pm', dict(recipient_ids=[1, 2], type='pm')),
+            ('1,2-group', dict(recipient_ids=[1, 2], type='group')),
```

Comment:
Is a 1-size group OK? Is pm a synonym for group?

Reply:
Oops, that needs correction. Thanks for pointing out.

Reply:
In principle that sounds likely - but pm and group appear to work in both cases on czo!